services/game-actions.js

```diff
@@ -0,0 +1,37 @@
+import firebase from 'firebase'
+import moment from 'moment';
+
+export function createGame(game) {
+  return new Promise((resolve, reject) => {
+    firebase.database().ref(`games/${game.id}`).set({
+      creator: "-LCF7qa2LI_oKKSQ3SIA",
```

Comment:
Make sure that you are passing a variable that holds the `user`'s `id`, and not hardcoding in a value.